framework/core/device.cpp

```diff
@@ -75,21 +75,6 @@ Device::Device(PhysicalDevice                        &gpu,
 	}
 
 	// Check extensions to enable Vma Dedicated Allocation
-	uint32_t device_extension_count;
-	VK_CHECK(vkEnumerateDeviceExtensionProperties(gpu.get_handle(), nullptr, &device_extension_count, nullptr));
-	device_extensions = std::vector<VkExtensionProperties>(device_extension_count);
-	VK_CHECK(vkEnumerateDeviceExtensionProperties(gpu.get_handle(), nullptr, &device_extension_count, device_extensions.data()));
-
-	// Display supported extensions
-	if (device_extensions.size() > 0)
-	{
-		LOGD("Device supports the following extensions:");
-		for (auto &extension : device_extensions)
-		{
-			LOGD("  \t{}", extension.extensionName);
-		}
-	}
-
```

Comment:
I think if you're going to move this to `PhysicalDevice` you should make the corresponding change to `HPPDevice` and `HPPPhysicalDevice` so that they stay mostly in sync.

Reply:
Addressed in newer commit